tests/tests/swfs/text/auto_size/return/output.txt

```diff
@@ -1,12 +1,12 @@
-150
+103
```

Comment:
was the old test output created wrongly?
It the new one equal to FP?

Reply:
That's correct. Somehow the `output.txt` wasn't from FP